app/views/shared/_intercom.erb

```diff
@@ -0,0 +1,4 @@
+<script type="text/javascript">
+	window.intercomSettings = { app_id: "<%=INTERCOM_APP_ID%>" };
```

Comment:
Where is INTERCOM_APP_ID defined?  Should that location be updated with a placeholder?